.github/scripts/setup_runners_service.sh

```diff
@@ -294,7 +294,7 @@ then
     docker run -d --name $DOCKER_NAME_KIBANA_NoSec --network="host" odfe-kibana-http:no-security
     docker ps
   else
-    sudo /usr/share/kibana/bin/kibana-plugin remove opendistro_security --allow-root
+    sudo /usr/share/kibana/bin/kibana-plugin remove opendistroSecurityKibana --allow-root
     sudo sed -i /^opendistro_security/d /etc/kibana/kibana.yml
```

Comment:
This has to be changed to sed -i /^opendistroSecurityKibana/d $KIBANA_ROOT/config/kibana.yml , as opendistro_security will no longer exist in the file.

Reply:
The settings have not changed. Just the plugin name